### PR TITLE
Fix missing uk core references in active profiles

### PR DESCRIPTION
**Pull request by APageNHS**
UKCore-Encounter still referencing Appointment
UKCore-MedicationDispense still referencing EpisodeOfCare

### CODE REVIEW

File: structuredefinitions/UKCore-Encounter.xml [hunk 1]
```diff
@@ -195,6 +195,13 @@
         <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
       </type>
     </element>
+    <element id="Encounter.appointment">
+      <path value="Encounter.appointment" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Appointment" />
```
Comment (APageNHS) [inline structuredefinitions/UKCore-Encounter.xml:202]: This creates a cyclic reference which currently breaks the GitHub validation

**Review by johngeo — COMMENTED** [review]
@APageNHS the changes look ok, it's just the cyclic referencing validation issue.  Do you want this approved, before  the validation issue is fixed?

**Comment by APageNHS** [comment]
> @APageNHS the changes look ok, it's just the cyclic referencing validation issue. Do you want this approved, before the validation issue is fixed?

Not yet, Created this PR more so that I don't forget about it - and to keep the validation issue on peoples radar

**Review by johngeo — APPROVED** [review]

File: structuredefinitions/UKCore-Encounter.xml (hunk 1 above)
Comment (APageNHS) [reply structuredefinitions/UKCore-Encounter.xml:202]: KevinM has added an exception, and CI validation now completes

**Review by SheenaSharma — APPROVED** [review]

**Pull request merged**